generators/app/index.js

```diff
@@ -486,13 +489,29 @@ module.exports = class extends Generator {
         apptype: this.props.apptype[0]
       })
 
-    // library package only (fixed)
-    if (this.props.apptype[1] === 'library') {
-      this.fs.copy(
-        this.templatePath('tests/xqs/**'),
-        this.destinationPath('content/')
-      )
-    }
+    this.fs.copyTpl(
+      this.templatePath('tests/xqs/test-suite.xql'),
+      this.destinationPath('test/xqs/test-suite.xql'), {
+        apptype: this.props.apptype[0],
+        short: this.props.short,
+        defcoll: this.props.defcoll,
+        defuri: this.props.defuri,
+        version: this.props.version,
+        author: this.props.author,
+        website: this.props.website,
+        title: this.props.title
+      })
+    this.fs.copyTpl(
+      this.templatePath('tests/xqs/test-runner.xq'),
+      this.destinationPath('test/xqs/test-runner.xq'), {
+        short: this.props.short,
+        defcoll: this.props.defcoll,
+        defuri: this.props.defuri,
+        version: this.props.version,
+        author: this.props.author,
+        title: this.props.title
+      })
+
     // all application packages (fixed)
     if (this.props.apptype[1] === 'application') {
```

Comment:
Could you explain the value of `this.props.apptype`? I am still confused

Reply:
S.a. https://github.com/eXist-db/generator-exist/blob/11c9736c90695b142ffd7d22a5d5ccba2e8dd9ac/generators/app/index.js#L60